packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,189 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/95870/
+  - title: "Migrate 'ThemeData.toggleableActiveColor' to individual themes"
```

Comment:
What is the difference between the migration options here? This title is shown in the IDE for the user to choose from (In the QuickFix pane), if they all have the same name it won't be clear what migration to pick.

Reply:
The difference is different ThemeData constructors 
In this PR, I've divided fixes for `ThemeData`, `ThemeData.copyWith` and `ThemeData.raw`

I can see titles are the same here for different constructors. 

```

  # Changes made in https://github.com/flutter/flutter/pull/41859
  - title: "Remove 'brightness'"
    date: 2020-12-10
    element:
      uris: [ 'cupertino.dart' ]
      constructor: ''
      inClass: 'CupertinoTextThemeData'
    changes:
      - kind: 'removeParameter'
        name: 'brightness'

  # Changes made in https://github.com/flutter/flutter/pull/41859
  - title: "Remove 'brightness'"
    date: 2020-12-10
    element:
      uris: [ 'cupertino.dart' ]
      method: 'copyWith'
      inClass: 'CupertinoTextThemeData'
    changes:
      - kind: 'removeParameter'
        name: 'brightness'
```

Reply:
Ok, I think we have gotten mixed up here somehow. If so I apologize. :) The ones you have referenced are not `bulkApply: false`.

If these are all unique methods, ['', 'raw', 'copyWith'], why are they `bulkApply: false` again? I thought there were more than one choice to migrate each?

For this one in particular ,`constructor: ''`, for the list of `changes`, is the user meant to pick one or apply all of them? Right now, this will apply all of the listed `changes`. If you mean for the user to pick one, they need to each be a different fix rule with `bulkApply: false`. The `title` represents one fix. so if there are optional choices, there should be a separate, unique `title` for each so the user can pick which one they want to apply. This currently will not present the user with choices, but apply all of the listed `changes` when the user picks it.

Reply:
That's my bad, I'm fairly new to dart fix. 

I think `bulkApply: false` is not required, I have updated the conditions and removed `bulkApply: false`.

Since this property is used by multiple widgets and we cannot detect which one of them is present in the code hence I am adding all the individual temes like this:

https://user-images.githubusercontent.com/48603081/166324057-3f347bc9-fa9f-412e-99f7-cb2d1ce669a1.mov

Reply:
When themes are already present, I am doing .copWith (similar to colorScheme fix) 

Uploading Screen Recording 2022-05-03 at 02.23.05.mov…

Reply:
Not your bad at all, this is a very complicated migration and I think you have provided a very helpful transformation. 

I see now. That is really nice, thank you very much, and apologies again for the mix up! Since they are not `bulkApply`, you can add tests for them in the `.expect` files.